### DISCUSSION TITLE
Virtual Environment cannot access machines available from host VM

### DISCUSSION THREAD

**Discussion by michelrtm** [actions/virtual-environments#5506] [General questions] [answered]
When running a Github Action on a self-hosted machine, if I try to connect to a network machine (which the host can access) I always get either a Connection timeout or a DNS resolution issue. I've tried changing the hosts file, resolv.conf files but nothing has fixed the issue. Is there any configuration so that the Virtual Environment uses the host's network, or am I missing something else?

**Comment by miketimofeev** [comment, marked as answer]
@michelrtm hi! No, we don't do anything special with the configuration. I think the better place to ask this is https://github.com/actions/runner as you use a self-hosted agent

**Reply by michelrtm** [reply]
Thanks for your answer @miketimofeev! I thought this kind of issue would be environment related, I'll ask it there.
